index.js

```diff
@@ -1,8 +1,147 @@
+
+    // Add card info Title
+    const cardInfoLiTitle = document.createElement('li');
+    const cardInfoTitle = document.createElement('p');
+    cardInfoTitle.classList.add('card-info-title');
+    cardInfoTitle.innerText = project.companyName;
+    cardInfoLiTitle.appendChild(cardInfoTitle);
+    cardInfo.appendChild(cardInfoLiTitle);
+
+    // Add dot
+    const cardInfoLiDotOne = document.createElement('li');
+    const imageCounterOne = document.createElement('img');
+    imageCounterOne.src = './img/counter.png';
+    imageCounterOne.alt = 'Counter';
+    cardInfoLiDotOne.appendChild(imageCounterOne);
+    cardInfo.appendChild(cardInfoLiDotOne);
+    // Add Author Position
+    const cardInfoLiPosition = document.createElement('li');
+    const projectPosition = document.createElement('p');
+    projectPosition.classList.add('author-skills');
+    projectPosition.innerText = project.projectPosition;
+    cardInfoLiPosition.appendChild(projectPosition);
+    cardInfo.appendChild(cardInfoLiPosition);
+
+    // Add dot
+    const cardInfoLiDotTwo = document.createElement('li');
+    const imageCounterTwo = document.createElement('img');
+    imageCounterTwo.src = './img/counter.png';
+    imageCounterTwo.alt = 'Counter';
+    cardInfoLiDotTwo.appendChild(imageCounterTwo);
+    cardInfo.appendChild(cardInfoLiDotTwo);
+
+    // Add Date
+    const cardInfoLiDate = document.createElement('li');
+    const projectDate = document.createElement('p');
+    projectDate.classList.add('card-date');
+    projectDate.innerText = project.projectDate;
+    cardInfoLiDate.appendChild(projectDate);
+    cardInfo.appendChild(cardInfoLiDate);
+    workCardRight.appendChild(cardInfo);
+
+    // Add Inner Info
+    const cardInnerInfo = document.createElement('p');
+    cardInnerInfo.classList.add('card-inner-info');
+    cardInnerInfo.innerText = project.projectDescription;
+    workCardRight.appendChild(cardInnerInfo);
+
+    // Add Skills
+    const skills = document.createElement('ul');
+    skills.classList.add('skills');
+    project.projectTechnologies.forEach((tech) => {
+      const techItem = document.createElement('li');
+      techItem.classList.add('card-skills-li');
+      techItem.innerText = tech;
+      skills.appendChild(techItem);
+    });
+
+    workCardRight.appendChild(skills);
+
+    // Add button
+    const seeMoreBtn = document.createElement('button');
+    seeMoreBtn.classList.add('see-project-button');
+    seeMoreBtn.innerText = 'See Project';
+    seeMoreBtn.type = 'button';
+    seeMoreBtn.dataset.key = project.key;
+    workCardRight.appendChild(seeMoreBtn);
+
+    projectArticle.appendChild(workCardRight);
+
+    cardGrid.appendChild(projectArticle);
+  });
+}
+
+implementProjects();
+
+// POP UP WINDOW
+const popupWindow = document.querySelector('.popup-window');
+const seeProjectBtns = [...document.querySelectorAll('.see-project-button')];
+const showPopupWindow = (btn) => {
+  popupWindow.style.display = 'flex';
```

Comment:
- [ ] Kindly keep all of the stylings in a stylesheet file, this will help you in the future when you want to do any update or debug!